src/shogun/regression/GLM.cpp

```diff
@@ -0,0 +1,24 @@
+#include <shogun/lib/config.h>
+
+#include <shogun/labels/RegressionLabels.h>
+#include <shogun/mathematics/Math.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/regression/GLM.h>
+#include <utility>
+using namespace shogun;
+
+GLM::GLM(): LinearMachine(){
+    init();
+}
+
+void GLM::init(){
+
+}
+GLM::GLM(DescendUpdaterWithCorrection* descend_updater, Family family, LinkFunction Link_fn, 
+	float64_t alpha, float64_t lambda): LinearMachine(){
+        m_alpha= alpha;
```

Comment:
nitpicking: plz be consistent with equation either `<variable>= value` or `<variable> = value`